torch/onnx/_internal/fx/export.py

```diff
@@ -21,9 +22,31 @@
 from torch.fx.passes import fake_tensor_prop
 from torch.nn.utils import stateless
 from torch.onnx import _constants, _type_utils
-from torch.onnx._internal import _beartype
+
+from torch.onnx._internal import _beartype, diagnostics
 from torch.utils import _pytree
 
+
+class _GetAttrCallable(Protocol):
+    def __call__(self, name: str) -> Any:
+        ...
+
+
+def _module_not_available(module_name: str, package_name: str) -> _GetAttrCallable:
+    def _inner(name: str):
+        raise RuntimeError(
+            f"{module_name} is not available. Please install {package_name} to use this feature."
+        )
+
+    return _inner
+
+
+def _fake_module_import(module_name: str, package_name: str) -> ModuleType:
```

Comment:
Should we open another file to store all utility functions? Internal functions is piling up. What do you think?

Reply:
lol I ended up deleting these functions, and resorting to removing the part in `verification.py` that introduces the dependency on `torch/onnx/_internal/fx/export.py`. 

I wasn't able to figure out how to 100% handle the import problem. There is always something not working great with type annotations.

Reply:
So right now the hope is, as long as `torch.onnx` doesn't import `torch.onnx._internal.fx`, we are good importing onnx and onnxscript directly under the folder.

Reply:
I also ran into that circular import before...